tavern/tomes/file_mkdir/metadata.yml

```diff
@@ -0,0 +1,14 @@
+name: Make Directory
+description: Will make a directory.
+author: nickobrien
+support_model: FIRST_PARTY
+tactic: RESOURCE_DEVELOPMENT
```

Comment:
I'd change this to EXECUTION
Resource development is more like setting up infra

Reply:
it was a meme thooooo 🥲

(but ok)